bc/channel/selectors.py

```diff
@@ -17,5 +18,47 @@ def get_mastodon_channel() -> Channel:
     return obj
 
 
-def get_enabled_channels() -> Iterable[Channel]:
+def get_all_enabled_channels() -> Iterable[Channel]:
```

Comment:
```suggestion
def get_all_enabled_channels() -> QuerySet[Channel]:
```

Reply:
Thanks for the suggestion. I'll change the type hint.